Makefile

```diff
@@ -1,5 +1,5 @@
 ifneq (,)
-.error This Makefile requires GNU Make.
+	.error This Makefile requires GNU Make.
```

Comment:
Are you sure this tab is valid? Keep in mind these 3 lines are BSD Makefile syntax, so that in case of using the wrong installed `make` command (e.g. on OSX), it will throw out this error.

Rest of the file is GNU.